internal/cli/atlas/deployments/setup.go

```diff
@@ -493,9 +501,17 @@ func (opts *SetupOpts) runConnectWith(cs string) error {
 	switch opts.connectWith {
 	case skipConnect:
 		fmt.Fprintln(os.Stderr, "connection skipped")
+	case compassConnect:
+		if !compass.Detect() {
+			return errCompassNotInstalled
+		}
+		if _, err := fmt.Fprintln(opts.OutWriter, "Launching MongoDB Compass..."); err != nil {
```

Comment:
this message should go to stderr

```suggestion
		if _, err := log.Warningln("Launching MongoDB Compass..."); err != nil {
```

Reply:
ok to address on a followup